src/components/home/AllCollections.js

```diff
@@ -0,0 +1,292 @@
+import { makeStyles, Button, IconButton } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import React, { useContext, useState } from 'react';
+import clsx from 'clsx';
+import Link from 'next/link';
+import { useTranslation } from 'react-i18next';
+import Drawer from '@material-ui/core/Drawer';
+import AppBar from '@material-ui/core/AppBar';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import List from '@material-ui/core/List';
+import Divider from '@material-ui/core/Divider';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import MenuOpenIcon from '@material-ui/icons/MenuOpen';
+import { useTheme } from '@material-ui/styles';
```

Comment:
You can use `import { useTheme } from '@material-ui/core/styles';` No need to install another package.